sklearn/naive_bayes.py

```diff
@@ -115,6 +115,12 @@ class GaussianNB(BaseNB):
 
     Read more in the :ref:`User Guide <gaussian_naive_bayes>`.
 
+    Parameters
+    ----------
+    class_prior : array-like, size=(n_classes,)
```

Comment:
is prior an array or a dict in LDA? it seems to me it should be a dict : class_name -> value (as we do for class_weight)